src/alpine/3.6/Dockerfile

```diff
@@ -36,7 +35,10 @@ RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
         userspace-rcu-dev \
         lttng-ust-dev
 
-# Currently apk cannot find lldb for 3.8, and this is not required for execution so leaving it commented out.
+# On 3.6, a recent version of llvm from edge/main must be used to install lldb.
 # Details: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/55 
-#RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/testing add --no-cache \
-#        lldb-dev
+RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
```

Comment:
Can you please add llvm to the previous "RUN apk …" instead? Each run creates a separate layer in the docker image and @MichaelSimons used to ask me to reduce number of layers in the past.

Reply:
The default repositories (v3.6/main and v3.6/community, used in the previous "RUN" command) don't have llvm5, which is why I put this in a separate command specifying edge/main.

Reply:
@sbomer at line 34, the `RUN apk -X` uses the same https://dl-cdn.alpinelinux.org/alpine/edge/main.

Reply:
Oops, missed that - thank you!